Testing/VelaptorTests/Content/AtlasLoaderTests.cs

```diff
@@ -277,24 +326,23 @@ public void Load_WhenAtlasJSONDataFileDoNotExist_ThrowsException()
     }
 
     [Fact]
-    public void Load_WhenAtlasImageDataFileDoNotExist_ThrowsException()
+    public void Load_WhenAtlasImageDataFileDoesNotExist_ThrowsException()
```

Comment:
Refactor unit test name

```suggestion
    public void Load_WhenAtlasImageFileDoesNotExist_ThrowsException()
```